src/components/PageElements/PageElements.tsx

```diff
@@ -44,12 +50,11 @@ const PageElements: React.FC<PageElementProps> = ({
   stages,
   canEdit = false,
   isConsolidation = false,
-  isReview = false,
+  reviewInfo,
```

Comment:
Rather than passing _both_ `isReview` and `reviewInfo` down through multiple levels, I just passed `reviewInfo` and deduce `isReview` when required (it's only in one place)

Reply:
Ok - that's better : )